rust/operator-binary/src/crd/mod.rs

```diff
@@ -130,6 +132,10 @@ pub mod versioned {
 
         #[fragment_attrs(serde(default))]
         pub resources: Resources<StorageConfig>,
+
+        /// This field controls which [ListenerClass](https://docs.stackable.tech/home/nightly/listener-operator/listenerclass.html) is used to expose the webserver.
+        #[serde(default = "default_listener_class")]
```

Comment:
It is not obvious to me, when to use which annotation. I would use the `fragment_attrs` annotation here:

```suggestion
        #[fragment_attrs(serde(default))]
```

The default is set in the `default_config` function below.

Then it is also not necessary to have `DEFAULT_LISTENER_CLASS` and `default_listener_class()`.